src/assets/cru-scss/_global-website-list.scss

```diff
@@ -1,15 +1,3 @@
-/***** TESTING *****/
-.globalWebsites--is-open {
-  background-image: url(https://www.cru.org/images/globalWebsites--huge.jpg) !important;
```

Comment:
@j2trumpet Does this background still need to be here?  It's not missing the image.